components/migrate/steps/SignLoader3.vue

```diff
@@ -73,37 +101,57 @@ const whichIcon = () => {
   return iconIdle
 }
 
-const { data } = useGraphql({
+const { data: getFromNfts } = useGraphql({
   queryName: 'nftIdListByCollection',
   clientName: from,
   variables: {
     id: fromCollectionId,
     search: [{ currentOwner_eq: fromAccountId }],
   },
 })
+const fromNfts = computed(() => (getFromNfts.value as NftIds)?.nfts)
+
+const nextNfts = ref()
+const checkNextNfts = async () => {
+  const { data } = await useAsyncQuery({
+    query: nftIdListByCollection,
+    variables: {
+      id: nextCollectionId.value,
+      search: [{ currentOwner_eq: accountId.value }],
+    },
+    clientId: urlPrefix.value,
+  })
+
+  nextNfts.value = (data.value as NftIds)?.nfts
+}
 
-// const burnItems = async (ids: string[]) => {
-//   await transaction(
-//     {
-//       interaction: NFTs.BURN_MULTIPLE,
-//       nftIds: ids,
-//       urlPrefix: from,
-//     },
-//     from,
-//   )
-//   updateSteps('step3-burn')
-// }
-
-// const burnCollection = async () => {
-//   if (fromCollectionId) {
-//     await transaction({
-//       interaction: Collections.DELETE,
-//       collectionId: fromCollectionId,
-//       urlPrefix: from,
-//     })
-//     updateSteps('step3-burn-collection')
-//   }
-// }
+const burnItems = async () => {
+  const ids = fromNfts.value?.map((item) => item.id) || []
+
+  await transaction(
+    {
+      interaction: NFTs.BURN_MULTIPLE,
+      nftIds: ids,
+      urlPrefix: from,
+    },
+    from,
+  )
+  updateSteps('step3-burn')
+}
+
+const burnCollection = async () => {
+  if (fromCollectionId) {
+    await transaction(
```

Comment:
When is transaction resolved? on block / finalized ?

Reply:
mostly will check on `status.value === TransactionStatus.Finalized` and based on `steps.value`